tutorials/algorithms/04_vqe_advanced.ipynb

```diff
@@ -159,7 +171,7 @@
    "source": [
     "Here we see that result was arrived at much more quickly where `optimizer_evals` is 10 versus 72 when it started from a random value when the initial point was not supplied (default of None).\n",
```

Comment:
> Here we see that result was arrived at much more quickly where `optimizer_evals` is 10 versus 72

the `10 versus 72` needs updating - it looks like `9 versus 65` now. I did look to see whether it was possible to substitute values derived from the code into the text in the past as I wondered if things might change, but that did not seem readily doable. Maybe since it has changed, and maybe will do again, perhaps we should print some message via the code to highlight the different rather than having this in a text section.